client/src/App.js

```diff
@@ -1,12 +1,24 @@
 import React from "react";
 import CssBaseline from "@material-ui/core/CssBaseline";
+
+import { Switch, Route } from 'react-router-dom';
+
+// View Imports
+import Homepage from "./views/Homepage/Homepage.view"
```

Comment:
`Homepage.view` not sure why you're including these sub names/identifiers. If you can make a good argument why we should do that, I would be okay with it, but it's just not a common React way of doing things. I'd like to be consistent throughout the codebase.

Reply:
Force of habit from work! We use JS not React but have a naming convention to follow. I will change this.